plugins/MultiSites/angularjs/site/site.controller.js

```diff
@@ -13,20 +13,24 @@
 
         $scope.period = piwik.period;
         $scope.date   = piwik.broadcast.getValueFromUrl('date');
+        $scope.dashboardUrl = dashboardUrl;
         $scope.sparklineImage = sparklineImage;
         $scope.website.label  = piwik.helper.htmlDecode($scope.website.label);
 
         this.getWebsite = function () {
             return $scope.website;
         };
 
-        function sparklineImage(website){
-            var append = '';
+        function tokenParam() {
             var token_auth = piwik.broadcast.getValueFromUrl('token_auth');
-            if (token_auth.length) {
-                append = '&token_auth=' + token_auth;
-            }
+            return token_auth.length ? '&token_auth=' + token_auth : '';
```

Comment:
I think here we can just use `piwik.token_auth`. Does that work as well? It should always be set too, since when logged in we have to send it on every HTTP request.

Reply:
I just noticed it was like that before... not sure if there was a reason for that.

Reply:
I see nevermind, I understand the reasoning now.